Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -16,6 +16,7 @@ class MessageDTO: NSManagedObject {
     @NSManaged var createdAt: Date
     @NSManaged var updatedAt: Date
     @NSManaged var deletedAt: Date?
+    @NSManaged var isHardDeleted: Bool
```

Comment:
isHardDeleted is marked as optional in the schema

Reply:
It is not optional, it has a default value of `false`